documentation/components/navigation/_navigation.pug

```diff
@@ -0,0 +1,13 @@
+mixin makeNavUL(items)
+  ul.nav
+    each val, index in items 
+      li
+        if (typeof(val) === 'array')
```

Comment:
Arrays are considered `objects`. You have to do `val instanceof Array` if you wanna make sure it's an array

Reply:
Well that's embarrassing